modules/oadp-multiple-bsl-use-case.adoc

```diff
@@ -0,0 +1,189 @@
+* You must have an application with a database deployed on a Red Hat OpenShift cluster.
+
+.Procedure
+
+. Create the first `Secret` for the AWS S3 storage provider with the default name by running the following command:
++
+[source,terminal]
+----
+$ oc create secret generic cloud-credentials -n openshift-adp --from-file cloud=<aws_credentials_file_name> # <1>
+----
+<1> Specify the name of the cloud credentials file for AWS S3.
+
+. Create the second `Secret` for MCG with a custom name by running the following command:
++
+[source,terminal]
+----
+$ oc create secret generic mcg-secret -n openshift-adp --from-file cloud=<MCG_credentials_file_name> # <1>
+----
+<1> Specify the name of the cloud credentials file for MCG. Note the name of the `mcg-secret` custom secret.
+
+. Configure the DPA with the two BSLs as shown in the following example.
++
+.Example DPA
+[source,yaml]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: two-bsl-dpa
+  namespace: openshift-adp  
+spec:
+  backupLocations:
+  - name: aws
+    velero:
+      config:
+        profile: default
+        region: <region_name> # <1>
+      credential:
+        key: cloud
+        name: cloud-credentials
+      default: true
+      objectStorage:
+        bucket: <bucket_name> # <2>
+        prefix: velero
+      provider: aws
+  - name: mcg
+    velero:
+      config:
+        insecureSkipTLSVerify: "true"
+        profile: noobaa
+        region: <region_name> # <3>
+        s3ForcePathStyle: "true"
+        s3Url: <s3_url> # <4>
+      credential:
+        key: cloud
+        name: mcg-secret # <5>
+      objectStorage:
+        bucket: <bucket_name_mcg> # <6>
+        prefix: velero
+      provider: aws
+  configuration:
+    nodeAgent:
+      enable: true
+      uploaderType: kopia
+    velero:
+      defaultPlugins:
+      - openshift
+      - aws
+----
+<1> Specify the AWS region for the bucket.
+<2> Specify the AWS S3 bucket name.
+<3> Specify the region, following the naming convention of the documentation of MCG.
+<4> Specify the URL of the S3 endpoint for MCG.
+<5> Specify the name of the custom secret for MCG storage.
+<6> Specify the MCG bucket name.
+
+. Create the DPA by running the following command:
++
+[source,terminal]
+----
+$ oc create -f <dpa_file_name> # <1>
+----
+<1> Specify the file name of the DPA you configured.
+
+. Verify that the DPA has reconciled by running the following command:
++
+[source,terminal]
+----
+$ oc get dpa -o yaml
+----
+
+. Verify that the BSLs are available by running the following command:
++
+[source,terminal]
+----
+$ oc get bsl
+----
++
+.Example output
+[source,terminal]
+----
+NAME   PHASE       LAST VALIDATED   AGE     DEFAULT
+aws    Available   5s               3m28s   true
+mcg    Available   5s               3m28s   
+----
+
+. Create a backup `CR` with the default BSL. In the following example, note that the `storageLocation` field is not specified in the backup `CR`.
```

Comment:
I don't think you need `CR` here if you are referring to the custom resource in general. I did a quick search of the repo and couldn't find any example where `CR` is placed in backticks.

Reply:
You could do something like `. Create a backup CR with default BSL such as <test-backup1> in the following example.` 

If this admonition is a note then it should probably be marked up in a `[NOTE]` annotation.

Reply:
Removed the backticks for CR and added a note. Thanks.